src/ui/components/search/searchcomponent.js

```diff
@@ -566,6 +566,16 @@ export default class SearchComponent extends Component {
       },
       onChange: () => {
         DOM.trigger(DOM.query(this._container, inputSelector), 'input');
+      },
+      onMount: () => {
+        const inputEl = DOM.query(this._container, inputSelector);
+        if (!inputEl) {
+          return;
+        }
+        const hasAutocompleteResults = this._autocomplete.getState('hasResults');
+        if (inputEl.getAttribute('aria-expanded') !== hasAutocompleteResults) {
+          inputEl.setAttribute('aria-expanded', hasAutocompleteResults);
+        }
```

Comment:
do we need to guard this? or can it just be
`inputEl.setAttribute('aria-expanded', !!hasAutocompleteResults);`

Reply:
I didn't want to do extra dom mutations, though I can remove the guard if you want?

Reply:
From some light testing it doesn't look like there's any harm removing the guard (I don't see the elements "flash" in dev tools or anything) but I thought it was a little safer?

Reply:
I see, if the guard here serves a purpose of avoiding extra dom mutations then that's fine! I just wasn't sure why we didn't directly set it before